Source/Core/Common/ChunkFile.h

```diff
@@ -229,6 +229,19 @@ class PointerWrap
     return current;
   }
 
+  // The caller needs to fill in val with the appropriate value later on.
+  [[nodiscard]] u8* GetPositionAndDoInt(u32& val)
```

Comment:
I don't understand why this takes a value parameter. In the current singular usage the value passed to it is otherwise unused, and you're supposed to overwrite it later anyway, so it seems pointless?

I think the function name could also better reflect the intended usage. Maybe something like, `u8* ReserveU32()` or `ReserveBytes(size_t byte_count)` or something like that?

Reply:
The only reason I passed a value into the function was just to match the convention of the other PointerWrap functions. However, I could just remove it.

Reply:
Since you're meant to overwrite the value in post anyway, I think it's better to remove it.